### PR TITLE
Getting Started Update

### PR DESCRIPTION
**Pull request by munoz0raul**
As discussed with @mike-scott and @Tim-Anderson. I did some updates in the GS to add token instructions as well as done some small improvements.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for a08872f1ed6d44c90e008b82c860a2d3dd842a17 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/848/docs/artifacts/html/index.html

File: source/getting-started/install-fioctl/index.rst [hunk 1]
```diff
@@ -123,7 +123,7 @@ manager; ``go get``:
 .. _gs-fioctl-post-install:
 
 Post-Install
-^^^^^^^^^^^^
+############
```
Comment (angolini) [inline source/getting-started/install-fioctl/index.rst:126]: Could you, please, also change this subtitle "Post-install" to something else?

The first time I saw it I thought I would not need this step as I has not following the manual installation.

I would replace "Post-Install" by "Authenticate your access" or "Authenticate fioctl" or "Login fioctl"

What do you think?

**Review by angolini — APPROVED** [review]
I like the leveling changes!

The other comment is only a suggestion

**Comment by doanac** [comment]
Docs for 41139fe61c6d4d6c49c63b40322c2ea965fab148 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/849/docs/artifacts/html/index.html

File: source/getting-started/install-fioctl/index.rst (hunk 1 above)
Comment (munoz0raul) [reply source/getting-started/install-fioctl/index.rst]: Done!

**Review by mike-scott — APPROVED** [review]
LGTM

**Review by ricardosalveti — APPROVED** [review]
LGTM, let's merge to get it included in v81.

**Comment by mike-scott** [comment]
@MatthewCroughan Have you taken a look at this one yet?

**Comment by mike-scott** [comment]
Woops @ricardosalveti fast with the merge :)

**Review by Tim-Anderson — APPROVED** [review]
LGTM

File: source/getting-started/git-config/index.rst [hunk 2]
```diff
@@ -9,10 +9,35 @@ as part of your :ref:`account creation <gs-signup>`. Once configured, ``git`` wi
...
+   :width: 900
+   :align: center
+
+   FoundriesFactory Settings
+
+Select the tab :guilabel:`Tokens` and create a new **Api Token** by clicking on 
+the :guilabel:`+ New Token`.
+Complete with a **Description** and the **Expiration date** and select :guilabel:`next`.
+
+For this tutorial, check the :guilabel:`Use for source code access` box and
```
[the remark this answers is not in the document]
Comment (mike-scott) [reply source/getting-started/git-config/index.rst:28]: @munoz0raul Can pick up this fix for the next PR round.

[the remark this answers is not in the document]
Comment (munoz0raul) [reply source/getting-started/git-config/index.rst:28]: @Tim-Anderson I will change it in the next PR.

**Pull request merged**